packages/nc-gui/components/smartsheet/Grid.vue

```diff
@@ -413,7 +413,9 @@ const expandedFormOnRowIdDlg = computed({
 provide(ReloadRowDataHookInj, reloadViewDataHook)
 
 // trigger initial data load in grid
-reloadViewDataHook.trigger()
+// reloadViewDataHook.trigger()
+
+watch(meta, () => reloadViewDataHook.trigger(), { immediate: true })
```

Comment:
With this, wouldn't we trigger the loading twice in a row when the view changes?

Further up in `Grid.vue` there's another watcher:

```ts
watch(
  () => view.value?.id,
  async (next, old) => {
    if (next && old && next !== old) {
      await loadData()
    }
  },
  { immediate: true },
)
```